### DISCUSSION TITLE
font-display and svg

### DISCUSSION THREAD

**Discussion by dibasdauliya** [KaTeX/KaTeX#3074] [General]
Hello there,
I want to add `font-display: swap;` for `KaTeX` font. I tried it in my stylesheet but I believe it was not applied to `KaTeX` font.
I ran the lighthouse report and it was giving me same issue. Can you please help me how can I add that property? (I've used cdn links for KaTeX's css)

And another question is, If I want to replace all the math in my site with `SVG` to avoid excessive DOM size and for accessibility by using  `RegEx` how can I do that? Is there any tool or method that I can convert many `KaTeX`  math to `SVG`?

Thank you.

**Comment by edemaine** [comment]
To use `font-display: swap`, you would need to use a local copy of `katex.css` (or `katex.min.css`) instead of CDN.  Then you could add the rule to every `@font-face` declaration (at the top of the file).

Although I understand that `font-display: swap` is considered good practice for the web in general, I would consider ignoring the error message for KaTeX.  The formulas are likely unreadable or at least ugly without the correct fonts, and it's probably not the main part of the page, so I think the default behavior is fine.
